nextflow.config

```diff
@@ -111,7 +154,12 @@ params {
 
     // Juicer Java paramaters
     juicer_jvm_params          = '-Xms512m -Xmx4096m'
-    juicer_tools_jar           = 'https://github.com/aidenlab/Juicebox/releases/download/v2.13.06/juicer_tools_2.13.06.jar'
+    juicer_tools_jar           = 'https://github.com/aidenlab/Juicebox/releases/download/v2.16.00/juicer_tools_2.16.00.jar'
+    juicer_box_jar             = 'https://github.com/aidenlab/Juicebox/releases/download/v2.16.00/juicebox_2.16.00.jar'
```

Comment:
Can't that be in `bin/` or somewhere else?

Reply:
Save as above. I think this can be upgraded to nearest release version. I will change the url to new version by next push.